core-web/libs/portlets/dot-content-drive/portlet/src/lib/dot-content-drive-shell/dot-content-drive-shell.component.ts

```diff
@@ -42,11 +42,17 @@ export class DotContentDriveShellComponent {
 
     readonly itemsEffect = effect(() => {
         const query = this.#store.$query();
+        const currentSite = this.#store.currentSite();
```

Comment:
Mark this to remember that needs to be grabber from the global store when available.

Reply:
It is already from the global store, We added that change on a older [PR](https://github.com/dotCMS/core/blob/baf649086092d14888ee5dbd98475d64537e4282/core-web/libs/portlets/dot-content-drive/portlet/src/lib/store/dot-content-drive.store.ts#L166)